citrination_client/models/prediction_result.py

```diff
@@ -31,4 +31,15 @@ def get_value(self, key):
         try:
             return self._values[key]
         except KeyError:
-            return None
\ No newline at end of file
+            return None
+
+    def all_responses(self):
```

Comment:
Can we call it something more generic, like `all_names` or `all_keys`?

Reply:
Yes!